test_unstructured_ingest/test-ingest-confluence2.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
maybe a docstring at the top of each test to explain the purpose (given that we have two for confluence)?

Reply:
Sounds good, now added with [736e729](https://github.com/Unstructured-IO/unstructured/pull/906/commits/736e729d069e792adfcd8c5c015739645036af17) 👍